css/newTab.css

```diff
@@ -497,7 +497,7 @@ input[type=button] {
   right: auto;
 }
 
-#newtab-customize-panel[open="true"] {
+#newtab-customize-panel[data-open="true"] {
```

Comment:
Nit: note that other "true" are not quoted. Not sure if it makes a difference, but good to be consistent.